support/diagnosing-oc-issues.adoc

```diff
@@ -0,0 +1,12 @@
+[id="diagnosing-oc-issues"]
+= Diagnosing OpenShift command-line interface ('oc') issues
```

Comment:
"oc" should be in backticks in normal headings, not single quotes. Several other places in the PR. (But see my other comment for the _topic_map.yml, which is handled differently)

Reply:
Resolved throughout the PR.